### DISCUSSION TITLE
build colors in tailwind.config into tailwind.css

### DISCUSSION THREAD

**Discussion by ricardo-fnd** [tailwindlabs/tailwindcss#10473] [Help]
Hi! I'm integrating tailwind with storybook. I would like to present all colors configured in `tailwind.config.js --> theme: { colors }` in a *color picker* in storybook.

The problem is, if I'm not mistaken, tailwind will only create classes into `tailwind.css` of those `colors` that are used in the project in a component. 

Is there a way to ignore this behavior and just create a class for every color in `tailwind.config.js` even if its not being used?
Thanks!

**Comment by ricardo-fnd** [comment]
Notice that this is the script I'm running to generate the css:
npx tailwindcss -i ./src/styles/tailwind-directives.css -o ./src/styles/tailwind.css

**Comment by mihir258** [comment]
Hello
Thanks for the info i will try to figure it out for more  .
[Dinar Updates
](https://apps.apple.com/us/app/dinar-guru-dinarguru-app/id1581089419)

**Comment by wongjn** [comment]
You could try [safelisting classes](https://tailwindcss.com/docs/content-configuration#safelisting-classes).

**Reply by ricardo-fnd** [reply]
I think this is it... I will give it a try!
thanks!
